### ISSUE TITLE
[Scheduler] Doesn't display events in a certain situation

### ISSUE THREAD

**Issue by yanmariomenev** [telerik/kendo-angular#3501]
**Describe the bug**
When the event has an array of resources and that array is larger than the Resource used in the Scheduler the events don't display in the Scheduler. If the array is smaller then one of the Resource shows its events and that seems to work correctly.

**To Reproduce**
open the following demo- https://stackblitz.com/edit/angular-gr5jxj-6lrzux?file=app%2Fapp.component.ts
and check the screen capture to reproduce- http://somup.com/crXXh50V1d

**Expected behavior**
To display the Resource if it matches the group.

**Comment by joharzmn** [comment]
Workaround until the issue get fixed: 

```typescript
// dynamically selected resourceIds array
const resourceIds = [1,2];

this.events = this.events.map((item) => {
  // filtering resourceId from event which is not included in selected resourceIds
  item.resourceId = (item.resourceId as number[]).filter(i => resourceIds.includes(i));
  return item;
});
```

**Comment by ipeshev** [comment]
Fix available in [latest dev version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/).

**Comment by gkarapeev** [comment]
Fix released in [v3.1.0](https://www.telerik.com/kendo-angular-ui/components/scheduler/changelog/#fa10e438-2f53-570a-a7a2-d74d6a6fa222)
